<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderinformation</class>
 <widget class="QDialog" name="orderinformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о заказе</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Имя: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>Номер телефона: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Адрес: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="text">
      <string>Дата: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>Время: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bouquetLabel">
     <property name="text">
      <string>Букет: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>Цена: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Статус заказа: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dopInfoLabel">
     <property name="text">
      <string>Дополнительная информация: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Отменить заказ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rejectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Отклонить заказ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="workFloristButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Приступить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="completeFloristButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Завершить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="workCourierButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Приступить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="completeCourierButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Завершить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>cancelOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>rejectOrder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workCourierButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>workCourier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workFloristButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>workFlorist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>completeCourierButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>completeCourier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>completeFloristButton</sender>
   <signal>clicked()</signal>
   <receiver>orderinformation</receiver>
   <slot>completeFlorist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelOrder()</slot>
  <slot>rejectOrder()</slot>
  <slot>workFlorist()</slot>
  <slot>completeFlorist()</slot>
  <slot>completeCourier()</slot>
  <slot>workCourier()</slot>
 </slots>
</ui>
